python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -63,31 +65,33 @@ def process_visit(self, visit_dir, out_dir=None, prefix='lsst', verbose=False):
         phosim_amp_files \
             = sorted(glob.glob(os.path.join(visit_dir, f'{prefix}_a_*')))
         amp_files = defaultdict(list)
+
         for item in phosim_amp_files:
             sensor_id = '_'.join(os.path.basename(item).split('_')[4:6])
             amp_files[sensor_id].append(item)
+
         if out_dir is None:
             tokens = os.path.basename(phosim_amp_files[0]).split('_')
             out_dir = 'v%07i-%s' % (int(tokens[2]), 'ugrizy'[int(tokens[3][1])])
+
         if not os.path.isdir(out_dir):
             os.mkdir(out_dir)
 
         for sensor_id in amp_files:
+            # print time spent repackaging
             if verbose:
                 sys.stdout.write(sensor_id + '  ')
             t0 = time.time()
-            # Skip already-processed and compressed files.
-            gzip_file = self.mef_filename(amp_files[sensor_id][0],
-                                          out_dir=out_dir) + '.gz'
-            if os.path.isfile(gzip_file):
-                continue
-            print("repackaging", gzip_file)
-            self.repackage(amp_files[sensor_id], out_dir=out_dir)
+
+            # repackage with the same function for corner or main raft sensors
+            self.repackage(amp_files[sensor_id], out_dir=out_dir,
+                           verbose=verbose)
+
             if verbose:
                 print(time.time() - t0)
                 sys.stdout.flush()
 
-    def repackage(self, phosim_amp_files, out_dir='.'):
+    def repackage(self, phosim_amp_files, out_dir='.', verbose=True):
```

Comment:
Please add the doc string for `out_dir` and `verbose`. Note they are "optional".

Reply:
I removed the verbose option since there isn't anything printed (there was earlier).